Content.Client/GameObjects/Components/ActionBlocking/CuffedComponent.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Robust.Client.Graphics;
+using Robust.Client.Interfaces.ResourceManagement;
+using Robust.Shared.IoC;
+using Robust.Shared.GameObjects;
+using Content.Shared.GameObjects.Components.ActionBlocking;
+using Content.Shared.Preferences.Appearance;
+using Robust.Client.GameObjects;
+using Robust.Shared.Utility;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Client.GameObjects.Components.ActionBlocking
+{
+    [RegisterComponent]
+    public class CuffedComponent : SharedCuffedComponent
+    {
+#pragma warning disable 0649
+        [Dependency] private readonly IResourceCache _resourceCache;
```

Comment:
```suggestion
        [Dependency] private readonly IResourceCache _resourceCache = default!;
```
so you can remove the #pragma